src/V3Options.h

```diff
@@ -493,8 +493,10 @@ class V3Options final {
     int buildJobs() const { return m_buildJobs; }
     int convergeLimit() const { return m_convergeLimit; }
     int coverageMaxWidth() const { return m_coverageMaxWidth; }
-    int dumpTree() const { return m_dumpTree; }
-    bool dumpTreeAddrids() const { return m_dumpTreeAddrids; }
+    bool dumpTreeAddrids() const {
+        const auto it = m_dumpLevel.find("tree-addrids");
+        return it != m_dumpLevel.end() && it->second > 0;
+    }
```

Comment:
This is a hot function.  It shouldn't look into an array.  I'd say you could use the static int = -1 trick, but it might end up called before the V3Options stuff.

Likewise please check if other arrays you've added are in hot usages.

Reply:
we have `bool() V3OPtionavailable` exactly to fix that race, see https://github.com/verilator/verilator/commit/add3811f46f4eb38cacd30ca7817160868da3e00 so will just use that